symfony/phpunit-bridge/4.1/manifest.json

```diff
@@ -2,6 +2,7 @@
     "copy-from-recipe": {
         ".env.test": ".env.test",
```

Comment:
It's not part of this PR, but the `.env.test` file contains:

```
KERNEL_CLASS=App\\Kernel
```

I believe we need a single slash - tests fail currently with `Class "App\\Kernel" doesn't exist`

Reply:
that's a BUG of Dotenv: this should work! (`source .env.test` to prove it)
worked around by using single quotes.

Reply:
that's a BUG of Dotenv: this should work! (`source .env.test` to prove it)
worked around by using single quotes.